thirdparty/openapi3/components/schemas/AuthenticationValue.yaml

```diff
@@ -3,6 +3,7 @@ anyOf:
   - $ref: '../../../../fspiop/v1_1/openapi3/components/schemas/OtpValue.yaml'
   - $ref: '../../../../fspiop/v1_1/openapi3/components/schemas/QRCODE.yaml'
   - $ref: ./U2FPinValue.yaml
+  - $ref: ./FIDOPublicKeyCredential.yaml
```

Comment:
FIDOPublicKeyCredential has a payload for attestation  (credential registration)
for assertion, we have little different payload:
```
{
    id: 'ADSUllKQmbqdGtpu4sjseh4cg2TxSvrbcHDTBsv4NSSX9...',
    rawId: ArrayBuffer(59),
    response: AuthenticatorAssertionResponse {
        authenticatorData: ArrayBuffer(191),
        clientDataJSON: ArrayBuffer(118),
        signature: ArrayBuffer(70),
        userHandle: ArrayBuffer(10),
    },
    type: 'public-key'
}
```

Reply:
there must be `signature` & `userHandler`

Reply:
Ok I'll split FIDOPublicKeyCredential into FIDOPublicKeyCredentialAttestation and FIDOPublicKeyCredentialAssertion and make the changes.